src/pat/autotoc/autotoc.js

```diff
@@ -54,39 +65,43 @@ export default Base.extend({
             if (!id || $("#" + id).length > 0) {
                 id = self.options.IDPrefix + self.name + "-" + i;
             }
+
+            const tabId = `${id}-tab`;
+            $(section).attr("id", id).attr("aria-labelledby", tabId);
+
             if (window.location.hash === "#" + id) {
-                activeId = id;
+                activeId = tabId;
             }
             if (activeId === null && $level.hasClass(self.options.classActiveName)) {
-                activeId = id;
+                activeId = tabId;
             }
             $level.data("navref", id);
-            const $nav = $("<a/>");
-            $nav.appendTo(self.$toc)
+
+            const $navItem = $("<li/>");
+            $navItem
+                .addClass("nav-item")
+                .attr("role", "presentation")
+                .appendTo(self.$toc);
+
+            const $nav = $("<button/>");
```

Comment:
I would not use `<button>` here ... according to the docs (https://getbootstrap.com/docs/5.3/components/navs-tabs/#tabs) we could leave them `a` tags ...